static/app/views/replays/replays.tsx

```diff
@@ -215,7 +215,13 @@ const StyledPageHeader = styled(PageHeader)`
 
 const StyledPageContent = styled(PageContent)`
   box-shadow: 0px 0px 1px ${p => p.theme.gray200};
-  background-color: ${p => p.theme.white};
+  background-color: ${p => p.theme.background};
+`;
+
+const StyledPanelTable = styled(PanelTable)`
+  @media (max-width: ${p => p.theme.breakpoints[1]}) {
+    grid-template-columns: repeat(3, minmax(0, 1fr));
```

Comment:
instead of `1fr` would this work better, at all breakpoints, with something like `grid-template-columns: auto max-content max-content;`?

Reply:
@Danondso @ryan953 

Here are some rough designs I quickly mocked up,

Thinking from the mobile first, here's how I see it.
- Remove the project column all together, until there's 'space'
- Probably assuming too much here, but id imagine that the grid column for the timestamp would be `max-content`or alternatively fixed width and the left column would be `1fr`.
- Ellipsis when text to keep everything on one line. (Dirty but works for now)  

Designs below.
https://www.figma.com/file/ULqGN1mpdvUkiWJ8PCuo0z/Index-v0-%5BSpecs%5D?node-id=200%3A4719

Reply:
awesome, that's much more concrete and specific!